frontend/src/layout/navigation-3000/sidepanel/panels/sidePanelDocsLogic.ts

```diff
@@ -0,0 +1,41 @@
+import { actions, kea, reducers, path, listeners, connect } from 'kea'
+import { SidePanelTab, sidePanelLogic } from '../sidePanelLogic'
+
+import type { sidePanelDocsLogicType } from './sidePanelDocsLogicType'
+
+const posthogComDomain = 'https://posthog.com'
```

Comment:
Nit: constants should be uppercase, e.g. `POSTHOG_COM_DOMAIN`… or `WEBSITE_DOMAIN`